src/misc/lv_lru.c

```diff
@@ -82,14 +81,15 @@ lv_lru_t * lv_lru_new(size_t cache_size, size_t average_length, lv_lru_free_t *
     cache->free_memory = cache_size;
     cache->total_memory = cache_size;
     cache->seed = time(NULL);
```

Comment:
`time` is not provided by every platforms, so let's use something more generic. 
```suggestion
    cache->seed = lv_rand();
```

Reply:
Thanks for your suggestion. I have updated it also naming convention.